### PR TITLE
fix(onboarding): show warning when trying to re-encrypt existing account

### LINKED ISSUE #5029: User cannot re-encrypt new keys
# Bug Report

## Description

This happens only when the App is in the state - 
logged out after being signed in previously and user having tried to enter a wrong pwd. 

In this situation, when the user tries to use the seed phrase to re-encrypt a new pwd, they find that the UI is stuck & they cannot re-encrypt the new keys when they hit `Create Pwd` button at the end of the Re-encrypt flow with the following log :

image

## Steps to reproduce 
- Sign in to an account
- Once signed in, go to `Settings` and click `Sign out & Quit` to sign out.
- Launch the app again and entera wrong Pwd to simulate user forgetting their Pwd
- Click the drop down arrow beside the account name
- Click the `Add another existing key` button to enter the existing saved seed phrase for this account (_**or another account**_)
- Hit `Re-encrypt your keys` button
- Enter the new Pwd and hit `Create Pwd`

#### Expected behaviour
User is able to create a new Pwd

#### Actual behaviour
https://user-images.githubusercontent.com/95302135/157882689-c8483702-26bf-4ad8-94ae-f4f67b7a0d9f.mp4

### Additional Information

- Status desktop version: Master

### PR DESCRIPTION
**Pull request by jrainville**
Fixes #5029

image

The behaviour will be improved with this next issue that will introduce a new status-go endpoint

### CODE REVIEW

File: ui/app/AppLayouts/Onboarding/views/ConfirmPasswordView.qml [hunk 1]
```diff
@@ -124,6 +125,35 @@ Page {
...
+                        //% "Login failed"
+                        importLoginError.title = qsTrId("login-failed")
+                        //% "Login failed. Please re-enter your password and try again."
+                        importLoginError.text = qsTrId("login-failed.-please-re-enter-your-password-and-try-again.")
+                    }
+                    importLoginError.open()
+                }
+            }
+
+            MessageDialog {
```
Comment (saledjenic) [inline ui/app/AppLayouts/Onboarding/views/ConfirmPasswordView.qml:145]: @jrainville don't we have some kind of Status styled message popup?

**Review by saledjenic — APPROVED** [review]

**Review by noeliaSD — APPROVED** [review]

**Review by iurimatias — CHANGES_REQUESTED** [review]
I'm not always getting the warning

**Comment by jrainville** [comment]
@iurimatias fixed. Now the warning appears on entering the seed phrase. I left the one on entering password as well as a double safety.
Requesting-review again because I had to change more files.

**Comment by abhi-status** [comment]
Cheers @jrainville !
I can now see this pop-up in both places.

**Review by abhi-status — APPROVED** [review]

**Comment by noeliaSD** [comment]
@jrainville just one curious think... From code I can see the text message is written correctly with spaces after commas but look at my screenshot.. it seems it has no commas there.. is it just only my perception? I promise I haven't altered the code to make the screenshot! haha

Screenshot 2022-03-29 at 16 39 09

**Comment by noeliaSD** [comment]
> @noeliaSD now sure why yours looks like that. On Linux is looks fine image
> 
> Anyway, when we actually have a design, it might be fine then.

It was on Mac, but yes, just it seemed curious to me!

**Review by noeliaSD — APPROVED** [review]

**Pull request merged**